packages/node_modules/@ciscospark/react-component-video/src/index.js

```diff
@@ -3,19 +3,28 @@ import classNames from 'classnames';
 
 import styles from './styles.css';
 
-export default function Video({src, audioMuted}) {
+export default function Video({src, srcObject, audioMuted}) {
   // Need autoPlay to start the video automagically
+
+  function getEl(el) {
+    if (el && srcObject) {
+      el.srcObject = srcObject;
+    }
+  }
+
```

Comment:
We have to do this because React strips out all non-whitelisted props. Currently `srcObject` is not allowed